### ISSUE TITLE
Example config for repo without application at the root level

### ISSUE THREAD

**Issue by benjamin-bergia** [ninenines/erlang.mk#661]
Hi, I recently started using erlang.mk and I am having a great time. I built some simple OTP applications and I would like to create a single release containing all my applications. Following https://erlang.mk/guide/deps.html#_repositories_with_no_application_at_the_root_level, here are the steps I followed:

1. Create a directory `bundle/`
2. Added `bundle/erlang.mk`
3. Created `bundle/Makefile` with the following in it:
```
PROJECT = bundle
DEPS = myfirstapp mysecondapp
dep_myfirstapp = cp ../myfirstapp
dep_mysecondapp = cp ../mysecondapp
include erlang,mk
```
4. Added `LOCAL_DEPS = mysecondapp` to the `Makefile` of `myfirstapp`
5. Run `make` and `make rel`

I get no errors at this point. All dependencies are fetched and built. Release are also built but each dependency seem to build its own release instead of one unique release containing everything.

Am I doing anything wrong ? 
Do you know any repository using erlang.mk and this kind of structure that I can look into ?

PS: Once again, great project, I am really enjoying my time working with erlang.mk :)

**Comment by essen** [comment]
It shouldn't build any dependency if there's no relx.config at the top level directory, so this is weird. Can you try creating a test in https://github.com/ninenines/erlang.mk/blob/master/test/plugin_relx.mk that reproduces the issue and send a PR? It would help. Also check the core_deps tests if you need to copy/paste a few things (like creating cp deps and such).

**Comment by benjamin-bergia** [comment]
@essen I will give it a try today

**Comment by benjamin-bergia** [comment]
So before starting to write my test, I looked at the tests in `plugin_relx.mk` and `core_deps.mk` as you mentioned. I noticed, that I was missing the `make bootstrap-rel`.

So basically the steps required to build a release without root level application are:
1. Create a directory
2. Fetch erlang.mk
3. Create a Makefile specifying the dependencies
4. **Run make bootstrap-rel**
5. Run `make` or `make run`

I tested this procedure on my project and it does work. After checking all the information are in the doc. 
Sorry about that. I will close the issue.
